baselines/fedprox/pyproject.toml

```diff
@@ -37,21 +37,23 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = ">=3.8.15, <3.12.0"
-flwr = "1.3.0"
-ray = "1.11.1"
+python = ">=3.10.0, <3.11.0"
```

Comment:
Do we want to exclude 3.11?

Reply:
is not useful if the best we can do to specify some (more complicated) packages such as `torch` need to pin the point to a specific wheel+python version. Maybe there is a better way of doing this, but i couldn't find one:
```
torch = { url = "https://download.pytorch.org/whl/cu117/torch-2.0.1%2Bcu117-cp310-cp310-linux_x86_64.whl"}
torchvision = { url = "https://download.pytorch.org/whl/cu117/torchvision-0.15.2%2Bcu117-cp310-cp310-linux_x86_64.whl"}
```

Reply:
Should we move this to another PR?

Reply:
yes. let's leave it as is currently in the `.toml` then?